core/src/wallet/tezos/explorers/ExternalTezosLikeBlockchainExplorer.cpp

```diff
@@ -371,19 +375,45 @@ namespace ledger {
                                                                    getRPCNodeEndpoint());
         }
 
+        Future<std::shared_ptr<BigInt>>
+        ExternalTezosLikeBlockchainExplorer::getTokenBalance(const std::string& accountAddress,
+                                                             const std::string& tokenAddress) const {
+            const auto parseNumbersAsString = true;
+            return _http->GET(fmt::format("/account/mainnet/{}", accountAddress), {}, BCD_API_ENDPOINT)
+                .json(parseNumbersAsString)
+                .mapPtr<BigInt>(getContext(),
+                    [=](const HttpRequest::JsonResult &result) {
+                       const auto &json = *std::get<1>(result);
+                       if ((!json.IsObject() ||
+                           !json.HasMember("tokens") ||
+                           !json["tokens"].IsArray()) && !json.IsString()) {
```

Comment:
why this condition: !json.IsString()

Reply:
I think I copied this from another place in the code, but you are right, it does not make much sense, I'll remove it